docs/source/conf.py

```diff
@@ -0,0 +1,56 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'kvikio'
+copyright = '2022, NVIDIA'
+author = 'NVIDIA'
+
+# The full version, including alpha/beta/rc tags
+release = '22.04.00'
```

Comment:
It would be useful to add the `update-version.sh` script below to ensure that we can keep this string up-to-date with each release.

https://github.com/rapidsai/cudf/blob/branch-22.04/ci/release/update-version.sh